documentation/vespa-quick-start-multinode-aws.html

```diff
@@ -74,13 +74,13 @@ <h2 id="install-vespa">Install and run Vespa</h2>
 our case we chose to use <samp>ip-172-1-1-1.us-east-2.compute.internal</samp></p>
 <pre>
 $ for host in $(cat hosts.txt); do \
-ssh -f $host "sudo bash aws_bootstrap.sh $(localhost)"; done
+ssh -f $host "sudo bash aws_bootstrap.sh $(hostname)"; done
```

Comment:
I think this might just have to be 'ip-172-1-1-1.us-east-2.compute.internal' instead of $(hostname) to make it explicit.